mods/d2k/mod.yaml

```diff
@@ -126,7 +126,7 @@ Translations:
 	d2k:languages/english.yaml
 
 LoadScreen: LogoStripeLoadScreen
-	Image: d2k:uibits/loadscreen.png
+	Image: ./mods/d2k/uibits/loadscreen.png
```

Comment:
For the same reason as the last 4 prs.  The FileSystem path resolving code has never supported the custom mod.yaml : shorthand.  #10703 restores the behaviour.